hosts/svc-hv01/network.nix

```diff
@@ -1,139 +1,86 @@
+        "sh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDMbUizElFyULDlpEE9XHWWOca4ZXepS18ljh4Fj4YnJOAs7sbYzzhfMUiD703FIgK5YObzOlheu/PBbwUOStgcmPDuRalZWLr+0kCUYERfjLHkgliFx96xEFw9dluvII6JpbzFI/uvkEkQ3ESKapRcYAuBTk2sRoit8za+HX9sLmMueqNtN4H92sFYYm1wWy3FFgz/NN+uTh7F5nmA7SrSS/fpbmugcgBdR/Zy1YwSA8Rl1pagEvgN9/qAnP7pssvXr9pTCUNxVSQ7FlTUOHmxzG16RybYRikgevQaHtFYvmS7AuRvRDlQWhHt1drREGOIwwZPXD1smfQAsvP66J85j9aeanZdoBoJcvvFNer3071QGmi+5NHDSiG+YvoWt7qgiKLF4lOfByzjdoRRSg01uuhdQLOHHt0hbfyGS6hx//1MtjiXTElXvOOiUJ6AqfCSwOTK+72W6VKhKYcO11+Ngym1dyF3TtVcoEYN3JpUdbNq+qctMzXFMGovPEEMh7s= melody"
+        "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHuSECgZffKGH56xoVzITe43IdRyYbAr3sef8TJOrGGH liske"
+        "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIK255EY8KUx5cMXSuoERXJSzVnkDUM+y8sMAVrRoDBnn marcel"
+        "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIE5BQ5JF5v+LisKXafxKQrKfwthVvWydMrr6BDJ2YyAg adb"
+        "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC6nI2MlJcERyDuOqCtW+2K/S5aR6wDAuyI6kSh8EgGvCSdp5TFTxSgUU8Vo78QTnL8vz0Vy1RYffwIUfNFNqRy8DX16UpQ/NYbWnpqoZHVYex4SIg5G7ZbofOIb0CPDt8ZqCk0GaRp2FprjO4QGkeKzPA3mxqAabtxn6/cGK+4FJE2NCRHhy1GR3TcDjOQrWmKeSnar7NI+oZZmhzaVl6+H3YVat/vlJLQXlyneDeQZ4dOUZanbF8lAajgl7pEu18auVW0Db99A9MAOEgvmMTyU2TFXTa+XY2ZmLIO8EWAkzQHpQu0U2i5ZD0Uj/ppYLIdY6aixXOJYylri2d1Nlil+3RUU16y2TeTwSycldDpfUtCRp+6taLfxz245x82HuU4UqHP6MsOMuk5qD1blYSXE+2gyTC4yTujxc6ykbNMwmyOxzkM+jDQ2DepboDKdBzuMnAwpQM/zjfXjultfiSM8e+ph95wpNz64yep6kUvJzgjLsD8sHAOFWByGxxJJ0yrj10jchZg8osZvQAnfnwtX8+/sT6NFW7Ikxpz1Q/PPMCmlviiUCyu7N4TNh3vuQaMlOf6Oe8+HirHg2nPeZ+pIuZG2B80CafcAlY795dPBv3tYC+gYPyJzDUe1QhTqhlS5vtY2l46Jj2ukRsE0FYSbDVyfn1HevS9/mcBxXJ8kQ== maurice"
+      ];
+    };
 
-        "40-bring-prj-up-bridges" = {
-          matchConfig.Name = "prj-*";
-          networkConfig = {
-            DHCP = "no";
-            LinkLocalAddressing = "no";
-            KeepConfiguration = "yes";
-          };
-        };
-      }
-      (builtins.listToAttrs (lib.flatten (
-        lib.mapAttrsToList
-          (name: value: [
-            {
-              name = "10-${bond_device_name}.${builtins.toString value.vlan}";
-              value = {
-                matchConfig.Name = "${bond_device_name}.${builtins.toString value.vlan}";
-                networkConfig.Bridge = value.bridge;
-              };
-            }
-          ])
-          config.dd-ix.nets
-      )))
-    ];
+    postCommands = ''
+      zpool import -a
+      echo "zfs load-key -a; killall zfs" >> /root/.profile
```

Comment:
What is this good for to load the keys whenever root spawns a session? Why is a suicide killall needed? :scream:

Reply:
Don't ask me ;D I've copied it from the NixOS Wiki in the beginning and I've been using it since then without problems.

https://wiki.nixos.org/wiki/ZFS#Unlock_encrypted_ZFS_via_SSH_on_boot

Reply:
Root spawn a session as soon as a user connects using ssh, that way the user is instantly presented with the password prompt. As soon as he confirms his input the session is closed automatically.